systers_portal/templates/account/logout.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+{% block title %} - Sign Out{% endblock %}
+{% block content %}
+  <div class="container">
+    <form class="logout form-logout" role="form" method="post" action="{% url 'account_logout' %}">
+      <h2>Sign Out{</h2>
```

Comment:
Hehe, "Sign Out{" :smiley: 

Fix that and squash the commits, so that you have only one commit and I'm pulling your contribution. Everything else is ok.